tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -86,10 +81,7 @@ def adjust_end(end: datetime.datetime, interval: int) -> datetime.datetime:
     return end
 
 
-@pytest.mark.xfail(
-    reason="Deprecated test. Will be faded out once SessionsReleaseHealthBackend will be removed."
-)
-class OrganizationSessionsEndpointTest(APITestCase, SnubaTestCase):
```

Comment:
All the tests in this class should stay as well.

Reply:
I reverted this change but I would have no idea how to fix these these tests 😭 

Currently `store_session` is inserting to sessions API endpoint, and these calls seem like they might need to be fanned out to metrics sets/distributions/counters with a very different payload shape as before.

I don't see any existing tests that actually write data to the real snuba backend.

Reply:
Neat, thanks!!